From tony.luck@intel.com  Thu Aug 12 13:41:16 2010
From: "Luck, Tony" <tony.luck@intel.com>
To: "Greg KH" <gregkh@suse.de>
Cc: linux-kernel@vger.kernel.org,
	"Samium Gromoff" <_deepfire@feelingofgreen.ru>,
	"Randy Dunlap" <randy.dunlap@oracle.com>,
	"KOSAKI Motohiro" <kosaki.motohiro@jp.fujitsu.com>,
	"Andrew Morton" <akpm@linux-foundation.org>
Subject: serial: print early console device address in hex
Date: Thu, 12 Aug 2010 12:16:43 -0700
Message-Id: <4c64489b11469c2a9e@agluck-desktop.sc.intel.com>

Device addresses are usually printed in hex.

Signed-off-by: Tony Luck <tony.luck@intel.com>
Cc: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/8250_early.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/serial/8250_early.c
+++ b/drivers/serial/8250_early.c
@@ -203,13 +203,13 @@ static int __init parse_options(struct e
 
 	if (mmio || mmio32)
 		printk(KERN_INFO
-		       "Early serial console at MMIO%s 0x%llu (options '%s')\n",
+		       "Early serial console at MMIO%s 0x%llx (options '%s')\n",
 			mmio32 ? "32" : "",
 			(unsigned long long)port->mapbase,
 			device->options);
 	else
 		printk(KERN_INFO
-		      "Early serial console at I/O port 0x%lu (options '%s')\n",
+		      "Early serial console at I/O port 0x%lx (options '%s')\n",
 			port->iobase,
 			device->options);
 
